services/QuillLMS/spec/models/concept_feedback_spec.rb

```diff
@@ -192,19 +270,64 @@
     end
   end
 
+  describe "#gengo_jobs" do
+    subject { concept_feedback.gengo_jobs }
+
+    let(:english_text) { create(:english_text)}
+    let!(:gengo_job) { create(:gengo_job, english_text:)}
+    let(:concept_feedback) { create(:concept_feedback) }
+    let!(:mapping) { create(:translation_mapping, english_text:, source: concept_feedback)}
+
+    it { expect(subject).to include(gengo_job)}
+  end
+
   describe "#fetch_translations!" do
-    context "there is a translated_text associated" do
-      let(:t1) { create(:translated_text)}
-      let(:t2) { create(:translated_text)}
+    context "there is a gengo_job associated" do
+      let(:t1) { create(:gengo_job)}
```

Comment:
Let's use full names instead of abbreviations for variables:  `translation1`,` translation2`